universal-application-tool-0.0.1/app/forms/NumberQuestionForm.java

```diff
@@ -0,0 +1,74 @@
+package forms;
+
+import java.util.OptionalLong;
+import services.Path;
+import services.question.types.NumberQuestionDefinition;
+import services.question.types.QuestionDefinitionBuilder;
+import services.question.types.QuestionType;
+
+public class NumberQuestionForm extends QuestionForm {
+  private OptionalLong min;
+  private OptionalLong max;
+
+  public NumberQuestionForm() {
+    super();
+    setQuestionType(QuestionType.NUMBER);
+    this.min = OptionalLong.empty();
+    this.max = OptionalLong.empty();
+  }
+
+  public NumberQuestionForm(NumberQuestionDefinition qd) {
+    super(qd);
+    setQuestionType(QuestionType.NUMBER);
+    this.min = OptionalLong.empty();
+    this.max = OptionalLong.empty();
```

Comment:
suggestion (blocking): should these be read from the question definition?

Reply:
Another great catch - thanks!

Reply:
Glad to at least see that the tests caught this mistake :)